### ISSUE TITLE
Enforcing SSL?

### ISSUE THREAD

**Issue by zerg000000** [oakes/Nightcoders.net#1]
Currently, http://nightcoders.net/ serving on http, https would make visitors feel more safe to use.

**Comment by oakes** [comment]
Yeah I actually made an attempt early on to enable SSL but I had trouble getting the boot-reload web socket connection to work (which is used to do hot code reloading). I plan on making another attempt pretty soon though.

**Comment by gamecubate** [comment]
On a totally unrelated note (I thought better to say this here than by adding noise by opening a new ticket)...

Great work, this!

I work full time with ClojureScript/Reagent and I keep rediscovering nightcoders.net every few months, whenever I feel a need to test something (reagent based, most times) out.

Thanks. Your work is appreciated.

**Comment by oakes** [comment]
That is so good to hear - - I don't even have traffic logs so I have no idea if anyone is using it! I haven't forgotten about the SSL issue by the way, just been distracted.

**Comment by gamecubate** [comment]
Off the top, you might want, in build.boot, to configure boot-reload to use the secure option. Something like this perhaps:

```
(deftask run []
  ...
 (reload :asset-path "public" :secure true)
 ...
```

Not sure though. Found [this](https://github.com/adzerk-oss/boot-reload/issues/75) in boot-reload's git issues. I hope it helps.
